browser/browser_context_options_test.go

```diff
@@ -40,3 +41,24 @@ func TestBrowserContextSetGeolocation(t *testing.T) {
 	assert.Equal(t, 2.0, opts.Geolocation.Longitude)
 	assert.Equal(t, 3.0, opts.Geolocation.Accuracy)
 }
+
+func TestBrowserContextDefaultOptions(t *testing.T) {
```

Comment:
I think i would feel more comfortable with this change with a test that ensures that the parsing/export/merge does indeed work with all the possible fields.

Reply:
That's a double-edged sword. We'd be testing Sobek's export behavior. On the other hand, it makes sense because we might set incorrect `js` tag values in the struct. I'll send another PR soon.